js/privatebin.js

```diff
@@ -362,6 +361,36 @@ jQuery.PrivateBin = (function($, RawDeflate) {
             }
         };
 
+        /**
+         * converts size in bytes to human readable string
+         *
+         * The string is expected to be optional digits, followed by a time.
+         * Supported times are: min, hour, day, month, year, never
+         * Examples: 5min, 13hour, never
+         *
+         * @name Helper.humanizeBytes
+         * @function
+         * @param  {number} bytes
+         * @param {decimals} decimals - optional
+         * @return {String}
+         */
+        me.humanizeBytes = function(bytes, decimals = 2) {
+            const sizes = [
+                I18n._('Bytes'), I18n._('KB'), I18n._('MB'), I18n._('GB'),
```

Comment:
Purely a suggestion, as this is fully functional in the current state: Keep the untranslated strings in the sizes array, then translate only the needed one in line 383 (`me.sprintf('0 %s', I18n._(sizes[0]));`). This makes the array more readable and avoids unnecessary translations.